scenarios/sslstream.benchmarks.yml

```diff
@@ -12,10 +12,10 @@ variables:
   tlsVersion: "1.3" # <1.2|1.3>
   serverCertSelection: "CertContext" # <Callback|Certificate|CertContext>
   clientCertSelection: "CertContext" # <Callback|Collection|CertContext>
-  allowTlsResume: true
+  allowTlsResume: false # can be true for .NET 8+ only, defaults to false for parity with older versions (and QUIC)

```

Comment:
TLS Resume is on by default for platforms where it is implemented (.NET 7+), IMO users don't explicitly disable it unless there is a problem so we should use it benchmarks I think. (btw it should affect only the handshake benchmarks)

Reply:
Ok, makes sense.